rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/importexport/ImportExportController.java

```diff
@@ -256,4 +256,24 @@ public ResponseEntity<RequestSummary> uploadComponentAttachment(
         RequestSummary requestSummary = importExportService.uploadComponentAttachment(sw360User, file, request);
         return ResponseEntity.ok(requestSummary);
     }
+
+    @Operation(
+            summary = "Export users as CSV.",
+            description = "Export all users in CSV format.",
+            tags = {"ImportExport"},
+            parameters = {
+                    @Parameter(name = "Accept", in = ParameterIn.HEADER, required = true),
+            }
+    )
+    @PreAuthorize("hasAuthority('WRITE')")
+    @GetMapping(value = IMPORTEXPORT_URL + "/downloadUsers")
```

Comment:
Please use new syntax

```suggestion
    @RequestMapping(value = IMPORTEXPORT_URL + "/downloadUsers", method = RequestMethod.GET, produces = { MediaType.TEXT_PLAIN })
```